content/systems/filecoin_nodes/clock/_index.md

```diff
@@ -27,34 +27,31 @@ The Filecoin system clock is used:
 
-- clients SHOULD query an NTP server (`pool.ntp.org` is recommended) on an hourly basis to adjust clock skew.
-  - We recommend one of the following:
-    - `pool.ntp.org` (can be catered to a [specific zone](https://www.ntppool.org/zone))
+- Nodes SHOULD run an NTP daemon (e.g. timesyncd, ntpd, chronyd) to keep their clocks synchronized to a reliable external clock.
+  - We recommend one or more of the following sources:
+    - **`pool.ntp.org`** (more on how to use an [NTP pool](https://www.ntppool.org/en/use.html))
     - `time.cloudflare.com:1234` (more on [Cloudflare time services](https://www.cloudflare.com/time/))
     - `time.google.com` (more on [Google Public NTP](https://developers.google.com/time))
     - `ntp-b.nist.gov` ([NIST](https://tf.nist.gov/tf-cgi/servers.cgi) servers require registration)
-  - We further recommend making 3 measurements in order to drop by using the network to drop outliers
-  - See how [go-ethereum does this](https://github.com/ethereum/go-ethereum/blob/master/p2p/discv5/ntp.go) for inspiration
-- clients MAY consider using cesium clocks instead for accurate synchrony within larger mining operations
+- Larger mining operations MAY consider using local NTP/PTP servers with GPS references and/or frequency-stable external clocks for improved timing.
 
-Mining operations have a strong incentive to prevent their clock from drifting ahead more than one epoch to keep their block submissions from being rejected.  Likewise they have an incentive to prevent their clocks from drifting behind more than one epoch to avoid partitioning themselves off from the synchronized nodes in the network.
+Mining operations have a strong incentive to prevent their clock skewing ahead more than one epoch to keep their block submissions from being rejected.  Likewise they have an incentive to prevent their clocks skewing behind more than one epoch to avoid partitioning themselves off from the synchronized nodes in the network.
 
 ## Future work
 
-If either of the above metrics show significant network skew over time, future versions of Filecoin may include potential timestamp/epoch correction periods at regular intervals.
+If continued network timing aberration is seen over time, future versions of Filecoin may include potential timestamp/epoch correction periods at regular intervals.
 
 When recoverying from exceptional chain halting outages (for example all implementations panic on a given block) the network can potentially opt for per-outage "dead zone" rules banning the authoring of blocks during the outage epochs to prevent attack vectors related to unmined epochs during chain restart.
 
 Future versions of the Filecoin protocol may use Verifiable Delay Functions (VDFs) to strongly enforce block time and fulfill this leader election requirement; we choose to explicitly assume clock synchrony until hardware VDF security has been proven more extensively.
```

Comment:
The adoption of drand introduces a strong dependency on a common external clock; it may make sense to reword (or delete) this section. Since I am not fully aware of the implications and constraints on FIL's side, I'm not the person to do it.